src/components/Video.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Video extends React.Component {
+	constructor () {
+		super();
+		this.state = {
+			VideoComponent: false,
+		};
+	}
+
+	componentDidMount () {
+		this.loadPlayer();
+	}
+
+	loadPlayer () {
+		require(['react-player'], (mod) => this.setState(() => ({ VideoComponent: mod.default })));
```

Comment:
Why do you need arrow function inside setState?

Reply:
this looks like a left-over of a refactoring, this was a functional setState before, no function needed here. will fix.